apps/costcontrol/test/marionette/lib/costcontrol.js

```diff
@@ -1,225 +0,0 @@
-/* global module */
```

Comment:
OMG! you are right! I did some tests regarding some cost control tests and wanted to remove them but not this file! will add it again